Makefile.in

```diff
@@ -0,0 +1,108 @@
+PROJECT := ulwgl
+
+OBJDIR := build
+
+PREFIX ?= /usr
+BINDIR := $(PREFIX)/bin
+LIBDIR := $(PREFIX)/lib
+DATADIR := $(PREFIX)/share
+
+DESTDIR ?=
+USERINSTALL ?= xfalse
+
+
+.PHONY: all
+all: reaper ulwgl ulwgl-launcher
+
+
+$(OBJDIR)/.build-ulwgl: | $(OBJDIR)
+	$(info :: Building ulwgl )
+	sed 's|##INSTALL_PATH##|$(DATADIR)/$(PROJECT)|g' ULWGL/ulwgl-run.in > $(OBJDIR)/ulwgl-run
+	touch $(@)
+
+.PHONY: ulwgl
+ulwgl: $(OBJDIR)/.build-ulwgl
+
+ulwgl-bin-install: ulwgl
+	install -d $(DESTDIR)$(DATADIR)/$(PROJECT)
+	install -Dm 755 $(OBJDIR)/$(<)-run $(DESTDIR)$(BINDIR)/ulwgl-run
+
+ulwgl-dist-install:
+	$(info :: Installing ulwgl )
+	install -d $(DESTDIR)$(DATADIR)/$(PROJECT)
+	install -Dm 644 ULWGL/ulwgl_consts.py    -t $(DESTDIR)$(DATADIR)/$(PROJECT)
+	install -Dm 644 ULWGL/ulwgl_dl_util.py   -t $(DESTDIR)$(DATADIR)/$(PROJECT)
+	install -Dm 644 ULWGL/ulwgl_log.py       -t $(DESTDIR)$(DATADIR)/$(PROJECT)
+	install -Dm 644 ULWGL/ulwgl_plugins.py   -t $(DESTDIR)$(DATADIR)/$(PROJECT)
+	install -Dm 755 ULWGL/ulwgl_run.py     	 -t $(DESTDIR)$(DATADIR)/$(PROJECT)
+	install -Dm 644 ULWGL/ulwgl_util.py      -t $(DESTDIR)$(DATADIR)/$(PROJECT)
+	install -Dm 644 ULWGL/ULWGL_VERSION.json -t $(DESTDIR)$(DATADIR)/$(PROJECT)
+
+# Install both dist and sh script target
+ulwgl-install: ulwgl-dist-install ulwgl-bin-install
+# Install dist only target
+#ulwgl-install: ulwgl-dist-install
+
+
+$(OBJDIR)/.build-ulwgl-launcher: | $(OBJDIR)
+	$(info :: Building ulwgl-launcher )
+	sed 's|##INSTALL_PATH##|$(DATADIR)/$(PROJECT)|g' ULWGL/ULWGL-Launcher/ulwgl-run.in > $(OBJDIR)/ulwgl-launcher-run
+	touch $(@)
```

Comment:
This creates the ulwgl-run file within the ULWGL-Launcher directory that is to be installed in `~/.local/share/Steam/compatibilitytools.d`, correct? 

If so, this is unnecessary because the script automatically creates a relative symbolic link to our directory -- ULWGL.

Reply:
It will also throw a runtime error indeed. I can disable this target a little further down and we can keep it around if the need arises.

Reply:
Would prefer disabling it as it's unnecessary